docs/configuration/behavior/index.mdx

```diff
@@ -3,29 +3,21 @@ title: Behavior
 slug: /configuration/behavior
 ---
 
-## Set/change token
-
-Since your token is sensitive 🤫 and you would not want to keep it in your repository, we recommend to set/change it using the `helm upgrade` command:
-
-```terminal
-helm upgrade -n datree datree-webhook datree-webhook/datree-admission-webhook --reuse-values --set datree.token=<your-token>
-```
-
-(simply replace `<your-token>` with your actual [token](https://app.datree.io/settings/token-management))
-
----
-
 ## Options
 
 The following options are available for configuring Datree's behavior:
 
 | **Option** | **Description** | **Type** |
 |---|---|---|
+| `enforce` | Block resources that fail the policy check from being applied to the cluster | boolean |
+| `clusterName` | The name of the cluster to be displayed in your dashboard | string |
+| `scanIntervalHours` | The number of hours between cluster scans | number |
 | `policy` | The name of the policy to check, e.g: "Argo" | string |
+| `existingSecret` | A Secret object containing a Datree token | Object |
```

Comment:
Perhaps we should add the existingSecret.name and existingSecret.key to the table?